service/administration/user.go

```diff
@@ -0,0 +1,471 @@
+	PersonalAccessToken *string       `json:"personalAccessToken,omitempty"`
+	Id                  *int          `json:"id,omitempty"`
+	GroupNames          []*string     `json:"groupNames,omitempty"`
+	Groups              []*Group      `json:"groups,omitempty"`
+	DisplayName         *string       `json:"displayName,omitempty"`
+	OrganizationId      *int          `json:"organizationId,omitempty"`
+
+	// forceSendFields is a list of field names (e.g. "Keys") to
+	// unconditionally include in API requests. By default, fields with
+	// empty values are omitted from API requests. However, any non-pointer,
+	// non-interface field appearing in ForceSendFields will be sent to the
+	// server regardless of whether the field is empty or not. This may be
+	// used to include empty fields in Patch requests.
+	forceSendFields []string
+
+	// nullFields is a list of field names (e.g. "Keys") to include in API
+	// requests with the JSON null value. By default, fields with empty
+	// values are omitted from API requests. However, any field with an
+	// empty value appearing in NullFields will be sent to the server as
+	// null. It is an error if a field in this list has a non-empty value.
+	// This may be used to include null fields in Patch requests.
+	nullFields []string
+}
+
+type UserPolicy struct {
+	PolicyId   *string  `json:"policyId,omitempty"`
+	PolicyName *string  `json:"policyName,omitempty"`
+	PolicyType *string  `json:"policyType,omitempty"`
+	AccountIds []string `json:"accountIds,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ProgrammaticUser struct {
+	Name        *string       `json:"name,omitempty"`
+	Description *string       `json:"description,omitempty"`
+	Policies    []*ProgPolicy `json:"policies,omitempty"`
+	Accounts    []*Account    `json:"accounts,omitempty"`
+	Token       *string       `json:"token,omitempty"`
+	ProgUserId  *string       `json:"id,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ProgPolicy struct {
+	PolicyId   *string  `json:"policyId,omitempty"`
+	AccountIds []string `json:"accountIds,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type Account struct {
+	Id   *string `json:"id,omitempty"`
+	Role *string `json:"role,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type Token struct {
+	Name            *string `json:"name,omitempty"`
+	CreatedAt       *string `json:"createdAt,omitempty"`
+	TokenId         *int    `json:"tokenId,omitempty"`
+	TokenLastDigits *string `json:"tokenLastDigits,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type Group struct {
+	Id          *string   `json:"id,omitempty"`
+	Name        *string   `json:"name,omitempty"`
+	PolicyNames []*string `json:"policyNames,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ListUsersInput struct{}
+
+type ListUsersOutput struct {
+	Users []*User `json:"users,omitempty"`
+}
+
```

Comment:
CreateUserInput and CreateProgrammaticUserInput missing

Reply:
Not needed we are directly using User and ProgramatticUser structs for the same